src/vmod_dynamic.c

```diff
@@ -100,8 +100,8 @@ static struct VSC_lck *lck_dir;
 
 static unsigned loadcnt = 0;
 
-static const struct gethdr_s HDR_REQ_HOST = { HDR_REQ, "\005Host:"};
-static const struct gethdr_s HDR_BEREQ_HOST = { HDR_BEREQ, "\005Host:"};
+static const struct gethdr_s HDR_REQ_HOST = { HDR_REQ, (void*)"\005Host:"};
```

Comment:
this is actually an adjustment required post VC 7.7, which is not related to the vcdk migration. I am going to address it separately.